be/src/runtime/local_tablets_channel.cpp

```diff
@@ -679,11 +679,59 @@ Status LocalTabletsChannel::_open_all_writers(const PTabletWriterOpenRequest& pa
     }
 
     _is_replicated_storage = params.is_replicated_storage();
+    std::shared_ptr<DeltaWriterOpenContext> context = std::make_shared<DeltaWriterOpenContext>();
```

Comment:
we can move the execution to thread pool in the brpc entrypoint in `backend_service.cpp` or `internal_service.cpp` just like other brpc interface, e.g. exec_plan_fragment ...